examples/imgui_impl_glfw.cmake

```diff
@@ -0,0 +1,14 @@
+if (NOT TARGET glfw)
+    find_package (GLFW3 QUIET)
```

Comment:
Why is `QUIET` specified here? It would be good to fail here loudly instead of failing quietly and user scratching their head because the example is not built for some reason.

Same applies to all find_package calls in this implementation.

Reply:
Look a bit further down. We craft our own error message based on whether or not `glfw` was brought in as a CMake submodule or not.

Reply:
I understand this PR is closed now anyway, but maybe one of you still has an optinion on this that [s]he cares to elaborate, because I use similar mechanisms in other projects:

Shouldn't the order be the opposite? First check, if the target is already  available (e.g. through add_subdirectory) and only if not, fall back to find_package. Assuming a library has the ability to use a local version of a dependency at all, I'd be extremely suprised, if if that local version got replaced, just because for some completely unrelated reason someone installed the dependency system wiede via a package manager.